server/provision_utility.py

```diff
@@ -225,4 +166,3 @@ def provisionDatabase():
     _provisionImages()
     _provisionSegmentationGroups()
     _provisionStudies()
-    _provisionDefaultFeatureset()
```

Comment:
It's not necessary for site operation (as the other magic collections are groups are), so having it in this provisioning script will cause the "Basic" featureset (which likely holds little clinical value) to be re-added to the database after every restart, even if an administrator tries to delete it. IMO, sample / demo data, if we choose to provide it, should be via an op-in mechanism.